rosdep/base.yaml

```
@@ -2814,6 +2814,13 @@ libgphoto-dev:
     precise: [libgphoto2-2-dev]
     raring: [libgphoto2-2-dev]
     saucy: [libgphoto2-6-dev]
+libgpiod-dev:
+  arch: [libgpiod]
+  debian: [libgpiod-dev]
+  fedora: [libgpiod-devel]
+  opensuse: [libgpiod]
+  rhel: [libgpiod-devel]
```

It looks like this package is only available in EPEL 8.
```suggestion
  rhel:
    '*': [libgpiod-devel]
    '7': null
```